src/Razor/src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/CodeWriter.cs

```diff
@@ -62,15 +73,40 @@ public string NewLine
             }
         }
 
-        // Internal for testing.
-        internal CodeWriter Indent(int size)
+        public CodeWriter Indent(int size)
```

Comment:
I made this public. Otherwise things in `Extensions.1_X` projects etc were complaining because we share `CodeWriterExtensions.cs` as a shared source and it uses this method.